services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+
+from db.models import Actor, Genre, Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+    return movies.distinct()
+
+
+def get_movie_by_id(movie_id: int) -> Movie | None:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Using `Movie.objects.get(id=movie_id)` will raise a `DoesNotExist` exception if no movie is found with the given ID. Consider using `Movie.objects.filter(id=movie_id).first()` to return `None` if no movie is found, which aligns with the function's return type hint of `Movie | None`.